src/tm/class-wpml-tm-page-builders-hooks.php

```diff
@@ -57,7 +57,13 @@ public function pro_translation_completed_action( $new_post_id, array $fields, s
 	public function adjust_translation_fields_filter( array $fields, $job ) {
 		$worker = $this->get_worker();
 
-		return $worker->adjust_translation_fields_filter( $fields, $job );
+		$fields = $worker->adjust_translation_fields_filter( $fields, $job );
+		foreach ( $fields as &$field ) {
+			if ( isset( $field['field_wrap_tag'] ) && $field['field_wrap_tag'] ) {
+				$field['title'] = $field['field_wrap_tag'];
```

Comment:
I am not sure we should completely replace the title because it can have other information.

Can't you just add the wrap information:

```php
$field['title'] .= ' (' . $field['field_wrap_tag'] . ')';
```

Reply:
I don't see tests for this.

Reply:
@strategio I have created test for it in wp-content/plugins/wpml-page-builders/tests/phpunit/tests/tm/test-wpml-tm-page-builders-hooks.php:89